README.asciidoc

```diff
@@ -1042,14 +1039,13 @@ GET /_search
 
 [NOTE]
 ================================
-The ``View in Console'' links will only work if the docs are viewed in web-browser mode, as follows:
+The "View in Console" links will only work if the docs are viewed in web-browser mode, as follows:
 
 [source,sh]
 ---------------
-build_docs -d my_doc.asciidoc --open <1> --web <2>
+build_docs -d my_doc.asciidoc --open <1>
```

Comment:
Asciidoctor doesn't support `<1>` inside the text. It has to be at the end. Also, `--web` isn't a thing any more.